services/gitea/6.0.1/defaults/kustomization.yaml

```diff
@@ -1,5 +1,4 @@
----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- cm.yaml
+  - cm.yaml
```

Comment:
Why? 🙂

Reply:
I think the automation is doing some reformatting but it'll probably just happen this first bump 😅